student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(age, classroom, **args)
+    super(age, **args)
```

Comment:
- [ ]  Let us modify our initialize like this 

 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)